### PR TITLE
Adding Magento services topic

### PR DESCRIPTION
**Pull request by erikmarr**
## Purpose of this pull request

This pull request (PR) adds information about Magento services so customers can connect to SaaS services. This content was migrated from devdocs (https://github.com/magento/devdocs/pull/6981)

## Affected documentation pages

- https://docs.magento.com/m2/ce/user_guide/configuration/services/saas.html

## Affected Magento editions

- [x] Open Source
- [x] Commerce
- [x] B2B

### CODE REVIEW

File: src/configuration/services/saas.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  - https://devdocs.magento.com/recommendations/configure.html
+---
+
+Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Services]({% link configuration/services.md %}) >  Magento Services
+
+Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
+
+When you create a [Magento account]({% link magento/magento-account.md %}), identified by a MageID, you can generate a Magento API key to connect to the SaaS services within Magento. To connect to Magento Commerce SaaS services, the API key needs to be generated by the license-holder. Then the API key can be passed the systems integrator or development team, who can then manage the SaaS projects and environments on behalf of the license-holder.
+
+Each MageID can have one SaaS project and within that SaaS project you can have one or more SaaS environments depending on the type of Magento license you have:
```
Comment (misha-kotov) [inline src/configuration/services/saas.md:13]: depending on whether or not you have a license - or something similar 
reason: open source is not a license "type", in that case you just don't have a license
the options below may not need to change, up to you, but I would list them as
- Magento Commerce
- Magento Open Source

File: src/configuration/services/saas.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+|Field|[Scope]({% link configuration/scope.md %})|Description|
+|--- |--- |--- |
+|SaaS Project|Global|Name of the SaaS project that groups all of your SaaS environments.|
+|SaaS Environment|Global|Lists of environment IDs and names in the specified SaaS Project.|
+
+## Create SaaS Environment
+
+To create a SaaS Environment, request the Magento API key from the Magento license holder for your store.
+
+1. In the Magento Admin, choose **Stores** > Settings > **Configuration** > **Services** > **Magento Services**.
```
Comment (leslietilling) [inline src/configuration/services/saas.md:46]: This is our standard

```suggestion
1. On the _Admin_ sidebar, go to  **Stores** > _Settings_ > **Configuration**.

1.  In the left panel, expand **Services** and choose **Magento Services**.
```

File: src/configuration/services/saas.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+1. In the **API Keys** section, paste your key value into the **Production Api Key** field regardless of whether you are in a production environment or non-production environment.
+
+1. Click **Save Config**.
+
+   When you save, if there are any SaaS projects associated with your API key, those projects will appear in the the **SaaS Project** field located in the **SaaS Environment** section.
+
+1. If there are not any SaaS projects created, click the **Request New Project** to display the **Project Name** field and enter a name for your SaaS project.
+
+   Magento creates a SaaS project and as many SaaS environments you are entitled to.
```
Comment (leslietilling) [inline src/configuration/services/saas.md:56]:
```suggestion
   Magento creates a SaaS project and the number of SaaS environments you are entitled to.
```

File: src/configuration/services/saas.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+1. Select the **SaaS Environment** to use for the current configuration of your Magento store.
+
+To change the SaaS project or environment names, click the **Rename this Project** or **Rename this Environment** respectively.
```
Comment (leslietilling) [inline src/configuration/services/saas.md:60]: Do we need to Save Config here?

File: src/configuration/services/saas.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+---
+title: Magento Services
+redirect_from:
+  - https://devdocs.magento.com/recommendations/configure.html
+---
+
+Stores > Settings > [Configuration]({% link stores/configuration.md %}) > [Services]({% link configuration/services.md %}) >  Magento Services
+
+Magento allows you to link your Magento store to a SaaS service through the Admin UI. By configuring and managing SaaS Environments, you can connect to services such as Product Recommendations.
```
[the remark this answers is not in the document]
Comment (misha-kotov) [reply src/configuration/services/saas.md:9]: I don't think so. "SaaS" can be used to describe how something is deployed, which in our case is a service

File: src/configuration/services/saas.md (hunk 4 above)
Comment (misha-kotov) [reply src/configuration/services/saas.md:60]: no; the page refreshes automatically.

**Comment by leslietilling** [comment]
If this is for 2.3.5, it should merge into `core-2.3.5-community`

**Comment by misha-kotov** [comment]
> If this is for 2.3.5, it should merge into `core-2.3.5-community`

it is for 2.3.5

File: src/configuration/services/saas.md (hunk 4 above)
Comment (leslietilling) [reply src/configuration/services/saas.md:60]: Usually in the Configuration in the Admin you need to click the Save Config button when you make a change to the settings. If not, then fine.

Comment (misha-kotov) [reply src/configuration/services/saas.md:60]: yep, usually you do, but not in this case

**Comment by erikmarr** [comment]
Closing this PR because it was targeted to the wrong branch. New PR:
https://github.com/magento/merchdocs/pull/450

**Pull request closed without merging**